Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml

```diff
@@ -33,7 +33,7 @@
 			<Grid x:Name="Home" Visibility="Collapsed">
 				<Border Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 					<StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
-						<FontIcon Glyph="&#xE80F;" />
+						<FontIcon Glyph="&#xE10F;" />
```

Comment:
can we set  Glyph as staticResource on the top of the page to be reausable??(this comment applies for the rest of the page you did the update :p)

Reply:
You should be able to do that. FontIcon::Glyph is just a string.
But, validate on uwp and one of uno targets to be sure.

Reply:
@kazo0

Reply:
You can create a new file, like FontIcons.xaml or whatever makes sense, similar to the [Path.xaml](https://github.com/unoplatform/Uno.Gallery/blob/master/Uno.Gallery/Uno.Gallery.UWP/Views/Styles/Path.xaml)